README.md

```diff
@@ -18,6 +18,8 @@ See [CONTRIBUTING.md](CONTRIBUTING.md).
   The webhook secret that GitHub signs the POSTed payloads with. This is created when the webhook is defined. The default is `hush-hush`.
 - **`TRAVIS_CI_TOKEN`**<br>
   For scripts that communicate with Travis CI. Your Travis token is visible on [yourprofile](https://travis-ci.org/profile) page, by clicking the "show token" link. Also See: https://blog.travis-ci.com/2013-01-28-token-token-token
+- **`JENKINS_SLAVE_IPS`**<br>
```

Comment:
I think we use `host` and `worker` in build to avoid the issue (so `worker` makes more sense here anyway).

https://github.com/nodejs/build/tree/master/ansible#getting-things-done